$OpenBSD: patch-gettext-runtime_m4_fcntl-o_m4,v 1.1 2010/07/03 03:23:22 naddy Exp $

Provide fallback for AC_USE_SYSTEM_EXTENSIONS which was introduced
in autoconf 2.60.

--- gettext-runtime/m4/fcntl-o.m4.orig	Thu Jul  1 07:44:09 2010
+++ gettext-runtime/m4/fcntl-o.m4	Thu Jul  1 07:44:43 2010
@@ -12,7 +12,9 @@ dnl Written by Paul Eggert.
 AC_DEFUN([gl_FCNTL_O_FLAGS],
 [
   dnl Persuade glibc <fcntl.h> to define O_NOATIME and O_NOFOLLOW.
-  AC_REQUIRE([AC_USE_SYSTEM_EXTENSIONS])
+  m4_ifdef([AC_USE_SYSTEM_EXTENSIONS],
+    [AC_REQUIRE([AC_USE_SYSTEM_EXTENSIONS])],
+    [AC_REQUIRE([AC_GNU_SOURCE])])
   AC_CACHE_CHECK([for working fcntl.h], [gl_cv_header_working_fcntl_h],
     [AC_RUN_IFELSE(
        [AC_LANG_PROGRAM(
